more-style/desktop-contact.css

```diff
@@ -14,7 +14,11 @@
   }
 
```

Comment:
The contact section on mobile view has a background image on the right side, but the project does not have one. Please add a background image.